.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+  pull_request:
+    branches:
+      # Branches from forks have the form 'user:branch-name' so we only run
+      # this job on pull_request events for branches that look like fork
+      # branches. Without this we would end up running this job twice for non
+      # forked PRs, once for the push and then once for opening the PR.
+    - '**:**'
```

Comment:
I don't know if it matters in Yaml, but is `-'**:**'` indented the same as `branches:` here?

Reply:
Yes, look better anyway, a linter I tested thought so aswell